articles/serializers.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any
+
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from taggit.serializers import TaggitSerializer, TagListSerializerField
```

Comment:
try to limit the amount of third party packages you use, especially if you can implement it easily, because it reduces your flexibility, if it is not actively maintained might cause security issues or even break  your 
code

Reply:
okay